react/components/SearchForm.js

```diff
@@ -23,6 +23,7 @@ class SearchForm extends Component {
 
     this.state = {
       isMyLocationButtonVisible: true,
+      address: this.props.address,
```

Comment:
is it possible for the `address` prop to change? if so, it might require some `getDerivedStateFromProps` or something of the sort